Airports/EGPB/Sector.txt

```diff
@@ -29,21 +32,17 @@ COORD:N059.17.25.967:E001.42.47.127
 COORD:N059.00.19.526:E001.37.55.318
 COORD:N059.00.00.009:E001.37.53.534
 
-;EGPB APP
 SECTORLINE:PBA_PDA
 DISPLAY:PDRAD:PDRAD:PBAPP
 DISPLAY:PBAPP:PDRAD:PBAPP
-DISPLAY:Scottish AC East:Scottish AC East:PBAPP
```

Comment:
Moray probably should also be informed if PBAPP or PDRAD are online

Reply:
Any thoughts on the above?

Reply:
Sorry, notifications not coming through by email for some reason. I will have a look at this before the next release.

Reply:
Ah, I shall address this now!

Reply:
I think the PDRAD lines shouldn't display to either Moray or Tyne, since the area it covers (see below screenshot) seems to be rather self explanatory. The traffic contained within is also entirely low level and of little relevance to Moray/Tyne. Circle sectorlines for EGPD positions display to Tyne, Moray and Montrose already.

![image](https://user-images.githubusercontent.com/12017284/61956057-41894500-afb4-11e9-8916-cdc0090189d7.png)

The PBAPP lines on the other hand, I think it would make sense to - as you say - display to Moray, so I've turned that on.

![image](https://user-images.githubusercontent.com/12017284/61956146-709fb680-afb4-11e9-9f4a-76477e5ab775.png)